### ISSUE TITLE
Questions in train the Caltech dataset

### ISSUE THREAD

**Issue by TTMRonald** [qqwweee/keras-yolo3#101]
Hi, your work is really good.
Recently, I was training a model on the Caltech dataset, and the training pre-training weight was yolo_weights.h5. However, I got 2 questions. _Please give me some advice._

**1.** In my three attempt, batch size is 64, 32 and 1. However, the train loss of the last batch of data and verification loss appeared NAN in every epoch. 

loss: [0.100600891][0.0554395244][0.000677854288][0.0444676131][1.59023784e-05][32443]
loss: [nan][nan][nan][nan][nan][129792]
loss: [nan][nan][nan][nan][nan][519168]
loss: [0.806849][0.21845597][0.044070445][0.544054747][0.000267857627][32427]
loss: [nan][nan][nan][nan][nan][129792]
loss: [nan][nan][nan][nan][nan][519168]
loss: [0.134574041][0.0277190916][0.000505827134][0.10634128][7.84544955e-06][32439]
loss: [nan][nan][nan][nan][nan][129792]
loss: [nan][nan][nan][nan][nan][519168]
1181/1181 [==============================] - 8104s 7s/step - loss: 2604.5672 - val_loss: nan

The data is disrupted in three attempts, batch size is 32 and 1. So the problem is not in a specific data. 

**2.** The initial loss was very big, which about 7000, although the loss droped quickly.

Epoch 1/30
loss: [0.891565502][0][0][0.891565502][0][32448]
loss: [1402.12244][0.556834579][0.0775082335][1400.45483][0.141585827][129751]
loss: [7042.36621][6.19032288][0.573532701][5631.9][1.57996607][519031]
   1/1181 [..............................] - ETA: 3:38:02 - loss: 7058.6250
loss: [0.937976658][0.0490695685][0.000876675709][0.840730309][0.047300078][32444]
loss: [1398.72144][0.472872466][0.0680791587][1397.1228][0.119744442][129755]
loss: [7028.54][5.35876894][0.529046714][5622.56934][1.36192155][519034]
   2/1181 [..............................] - ETA: 1:58:05 - loss: 7051.7117

Thank you and regards.

**Comment by qqwweee** [comment]
1. no idea. What was your code version or did you modify any codes?
2. As you can see, the most of loss is confidence loss and it goes down quickly. It is OK.

**Comment by TTMRonald** [comment]
Thanks Reply！
I was using the code version that was 10 days ago. I was only modified anchors and aspect ratio of input ground truth.

**Comment by qqwweee** [comment]
@TTMRonald Sorry but I cannot tell the reason. The nan problem is fixed in b7cbd24.
